src/pages/index.js

```diff
@@ -73,35 +107,137 @@ function handleImageClick(title, link) {
 }
 
 function makeCard(cardData) {
-  const card = new Card(cardData, cardTemplate, handleImageClick);
+  const card = new Card(
+    cardData,
+    cardTemplate,
+    handleImageClick,
+    handleDeleteClick,
+    handleCardLike
+  );
   return card.generateCard();
 }
 
+function handleDeleteClick(cardId) {
+  console.log(cardId);
+  deleteConfirmModal.open(cardId);
+}
+
+function changeLike(card) {
+  card.querySelector(".heart-button").classList.toggle("heart-button_active");
```

Comment:
Now only class `Card` knows every **card selector**, so you need to encapsulate the likes logics inside its methods and call them in `index.js`. It’s a violation of **Single responsibility** principle